src/rules/pylint/mod.rs

```diff
@@ -20,23 +20,23 @@ mod tests {
     #[test_case(Rule::NonlocalWithoutBinding, Path::new("nonlocal_without_binding.py"); "PLE0117")]
     #[test_case(Rule::UsedPriorGlobalDeclaration, Path::new("used_prior_global_declaration.py"); "PLE0118")]
     #[test_case(Rule::AwaitOutsideAsync, Path::new("await_outside_async.py"); "PLE1142")]
-    #[test_case(Rule::ConstantComparison, Path::new("constant_comparison.py"); "PLR0133")]
+    #[test_case(Rule::ComparisonOfConstant, Path::new("comparison_of_constant.py"); "PLR0133")]
```

Comment:
I think we need to rename the files as well -- they're in `./resources/test/fixtures/pylint`.

Reply:
Just LMK if you have any trouble with this -- you'll probably need to rename these files, then run `cargo test --lib`, then `cargo insta review`, _then_ `cargo insta test --delete-unreferenced-snapshots` :joy:

Reply:
Huhu, thank you, I missed the last one 😄 The commands are incredibly slow on my machine and I need to rebuild on each commit, is there any way to make it faster ? Maybe it's because I don't have enough disk space, only python was supposed to run on this machine originally.

Reply:
Unfortunately there's no trick to making the rebuilds faster. Some of it is just Rust, but a lot of it is that we have a bad project structure right now -- we need to split the main crate into multiple crates, which should make compilation a lot faster (#1820).